documentation/URL_SPEC.md

```diff
@@ -0,0 +1,157 @@
+# Vol-E App URL Specification
+
+When used as a standalone app, Vol-E allows you to specify view settings via
+query parameters in the URL. This document provides an overview of the parameter
+specification.
+
+You can include none or all of these parameters in the URL. If a parameter is
+not specified, the listed default value will be used instead.
+
+If you are using our public build, set
+[`https://volumeviewer.allencell.org/viewer`](https://volumeviewer.allencell.org/viewer)
+as the base address. If you are running Vol-E locally, you can substitute this
+for a `localhost` address.
+
+## Common Examples
+
+| Query Parameters                            | Description                                                                                                                               |
+| ------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| `?url={url}&c0=ven:1&c2=ven:1,clz:1&view=Z` | Enable the first and third channel volumes and apply colorizing to the third. View in XY / Z-slice mode.                                  |
+| `?url={url}&c0=ven:1,rmp:15:180`            | Enables the volume of the first channel and ramps opacity for volume intensity values from 0% at histogram bin `15` to 100% at bin `180`. |
+| `?url={url}&reg=0:1,0:1,0.25:0.75,t=40`     | Clips along the Z-axis to the middle 50% of the volume, and sets the time to frame 40.                                                    |
```

Comment:
Any other common examples I should add? I use this mostly as a cookbook for users who don't want to read the whole spec.